package.json

```diff
@@ -35,7 +35,7 @@
     "build:compile": "babel --source-maps -d lib src",
     "build:bundle": "NODE_ENV=production webpack -p --progress",
     "build:bundle:dev": "webpack --optimize-occurence-order --progress",
-    "build:electron": "npm run clean && npm run build && cpx electron/config.json webapp/ && build -lwm",
+    "build:electron": "rimraf electron/dist && npm run clean && npm run build && build -wm --ia32 --x64 && scripts/check-electron.sh",
```

Comment:
There is now a linux build (deb). What this means is we get both 32 and 64 bit builds for windows and linux, but not mac because mac went 64 bit ages ago.